src/components/App/App.js

```diff
@@ -9,10 +9,17 @@ export default class App extends Component {
     super()
     this.state = {
       movies: [],
-      error: false
+      error: false,
+      search: ''
     }
   }
 
+  updateSearch = searchTerm => {
+    this.setState({
+      search: searchTerm
+    })
```

Comment:
We may want to go through and make sure we have semi-colons everywhere we intend.